s3manifestcollectstatic/management/commands/s3manifestcollectstatic.py

```diff
@@ -33,17 +34,25 @@ def handle(self, *args, **options):
 
         with TemporaryDirectory() as tmpdirname:
             saved_static_root = settings.STATIC_ROOT
-            saved_storage = settings.STATICFILES_STORAGE
+
+            if VERSION[0] >= 4 and VERSION[1] >= 2:
```

Comment:
`STORAGES` is not yet mandatory in Django 4.2 (just mutually exclusive with the old `STATICFILES_STORAGE`) so these might be better served with some getattrs.